<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="courseListWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>courseListWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>clickItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addItem()</slot>
  <slot>clickItem()</slot>
 </slots>
</ui>
